lassondeDB/urls.py

```diff
@@ -16,11 +16,12 @@
 from django.contrib import admin
 from django.urls import include, path
 
-from projects.views import homepage, loginUser, logoutUser
+from authentication.views import homepage, loginUser, logoutUser, registerUser
 
 urlpatterns = [
     path('', homepage, name='homepage'),
     path('login/', loginUser, name='login'),
+    path('register/', registerUser, name='register'),
     path('logout/', logoutUser, name='logout'),
```

Comment:
I would recommend moving these into an authentication specific urls file similar to the other apps

Reply:
These are the main URLS for the Django. As in the main application where all the URLs are registered.

Reply:
Oh i see what you mean, yeah can get that done in the next update